bluesky_queueserver/manager/tests/_common.py

```diff
@@ -25,14 +25,14 @@ def copy_default_profile_collection(tmp_path, *, copy_yaml=True):
     Returns the new temporary directory.
     """
     # Default path
-    pc_path = get_default_profile_collection_dir()
+    pc_path = get_default_startup_dir()
     # New path
     new_pc_path = os.path.join(tmp_path, "startup")
 
     os.makedirs(new_pc_path, exist_ok=True)
 
     # Copy simulated profile collection (only .py files)
-    patterns = ["[0-9][0-9]*.py"]
+    patterns = ["*.py"]
```

Comment:
This will need to be updated to include `*.ipy` as well, as noted above.

Reply:
Changed in all instances.